docs/semgrep-code/getting-started.md

```diff
@@ -29,333 +28,146 @@ import PlatformSigninGithub from "/src/components/procedure/_platform-signin-git
-
-<SemgrepScan />
-
-Semgrep Code is transparent: you can fully configure what rules are run and inspect the Semgrep syntax to understand how the finding was detected. The content of a rule can be customized to improve the true positive rate of a rule or its message to fellow developers.
-
-This document provides steps to get started with Semgrep Code for all tiers except where indicated.
-
-## Semgrep Code with Semgrep Cloud Platform
-
-Semgrep Code includes Semgrep Cloud Platform (SCP), a web application that helps security teams meet their goals, such as:
-
-* Sorting, filtering, triaging, and remediating security issues.
-* Enforcing coding standards through the creation of custom rules.
-* Preventing insecure code from reaching production or staging servers by blocking pull or merge requests, based on how you configure your rules.
-* Notifying security teams of findings (results) as well as communicating with other developers by leaving pull or merge request comments in GitHub, GitLab, or BitBucket.
-
-:::tip Suggested workflows 
-* You can use Semgrep Cloud Platform to scan remote repositories (GitHub, GitLab, or BitBucket) and consolidate the findings.
-* You can also use Semgrep Cloud Platform to consolidate findings from a Semgrep CLI scan performed on a **local** machine.
-* Semgrep Code can be integrated into your custom infrastructure without SCP. See [API](/semgrep-cloud-platform/semgrep-api) for details.
-:::
-
-### Scanning a repository
-
-#### Signing in to Semgrep Cloud Platform
-
-<PlatformSigninIntro />
+# Scan your codebase
 
-<Tabs
-    defaultValue="signin-github"
-    values={[
-    {label: 'Sign in with GitHub', value: 'signin-github'},
-    {label: 'Sign in with GitLab', value: 'signin-gitlab'},
-    ]}
->
+Semgrep Code is a static application security testing (SAST) tool that scans your repository for security vulnerabilities. You can scan your repository by integrating Code into your CI/CD pipeline, or you can scan repositories available locally on your machine.
 
-<TabItem value='signin-github'>
+## Use the Semgrep Cloud Platform (SCP) to automate the continuous scanning of your code
 
-<PlatformSigninGithub />
-
-See [Permissions in GitHub](/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab) to learn more about how Semgrep features use requested permissions in GitHub.
-
-</TabItem>
-
-<TabItem value='signin-gitlab'>
-
-<PlatformSigninGitlab />
-
-See [Permissions in GitLab](/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab) to learn more about how Semgrep features use requested permissions in GitLab.
-
-</TabItem>
-
-</Tabs>
+<PlatformAddRepo />
 
-:::tip
-To use Semgrep with your team, [create an organization account](/semgrep-cloud-platform/user-management/). An organization account enables users to share rules and perform triage or remediation as a team.
+:::tip Scan a demo application
```

Comment:
I think we can remove this admonition and let the user explore on their own. The demo feature is self-explanatory.